ocs_ci/ocs/ocp.py

```diff
@@ -133,8 +133,9 @@ def create(self, yaml_file=None, resource_name='', out_yaml_format=True):
             command += f"{self.kind} {resource_name}"
         if out_yaml_format:
             command += " -o yaml"
-
-        return self.exec_oc_cmd(command)
+        output = self.exec_oc_cmd(command)
+        logging.info(f"{yaml.dump(output)}")
```

Comment:
can this be debug?

Reply:
+1
Printing the entire yaml will flood the log for info

Reply:
done